apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivity.java

```diff
@@ -80,7 +80,7 @@ public boolean isEnabled() {
                 @Override
                 public void setEnabled(boolean enabled) {
                     Crashes.setEnabled(enabled);
-                    crashesEnabledPreference.setChecked(Analytics.isEnabled());
+                    crashesEnabledPreference.setChecked(Crashes.isEnabled());
```

Comment:
The same change will have different commit IDs. If it doesn't matter, I don't care. 😄